eclide/RepositoryFrame.h

```diff
@@ -60,8 +60,6 @@ class CRepositoryFrameT :	public CRepositorySlotImpl,
     {
         T * pT = static_cast<T*>(this);
         ContextState state;
-        if (s.mods.size() + s.attrs.size() == 0)
-            return;
```

Comment:
@dehilsterlexis Adding a "module" to std/plugins/bundle should not be permitted.